tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsWriter.cs

```diff
@@ -22,15 +22,16 @@ internal class DataStreamsWriter : IDataStreamsWriter
 {
     private static readonly IDatadogLogger Log = DatadogLogging.GetLoggerFor<DataStreamsWriter>();
 
-    private readonly BoundedConcurrentQueue<StatsPoint> _buffer = new(queueLimit: 10_000);
-    private readonly BoundedConcurrentQueue<BacklogPoint> _backlogBuffer = new(queueLimit: 10_000);
```

Comment:
[coding style] Can you revert this two lines? 😅  We like having the parameter name (`queueLimit`) in cases like this with no variable names so it's obvious what the `10_000` means when reading the code.